src/strings/PowerFxResources.en-US.resx

```diff
@@ -2398,6 +2398,9 @@
     <value>Invalid json pointer.</value>
     <comment>Generic json pointer parsing error.</comment>
   </data>
+  <data name="None" xml:space="preserve">
+    <value>None</value>
```

Comment:
>None [](http://example.com/codeflow?start=11&length=4)

Please add comments explaining the context in which this value will be displayed to the user (and preferably to the other types as well). Without them the translators don't have any context on what the term means, which will cause bad translations (e.g., wrong gender, wrong syntactical category, ...)

Reply:
For example, this is being used as a function category, which should be a feminine noun in Portuguese, but the current translation for the term Logical (2 down) is a masculine adjective.